Misc/NEWS.d/next/Core_and_Builtins/2025-04-26-08-49-05.gh-issue-132952.ifvP10.rst

```diff
@@ -0,0 +1,2 @@
+Speed up startup with the ``-S`` argument by about 19% by importing the
```

Comment:
it is worth mentioning "vs what" when stating an improvement %.

Reply:
It's 19% vs. before this commit, but it's only 1.5% faster than 3.13.0 (which is probably what most users care about, since they don't run `main`).  I would argue 1.5% is in the noise, so we probably shouldn't have a what's new entry at all -- this is just counteracting a regression in 3.14 development.

Reply:
It's only a NEWS entry, not text in the What's New, so I'll remove the 19% claim but leave the NEWS entry since this could affect behavior some users care about.